lib/ssl/src/ssl_handshake.erl

```diff
@@ -1121,17 +1124,21 @@ server_select_cert_key_pair_and_params(CipherSuites, [#{private_key := Key, cert
             end
     end.
 
-is_acceptable_cert(Cert, HashSigns, {Major, Minor}) when Major == 3,
-                                                         Minor >= 3 ->
+is_acceptable_cert(Cert, HashSigns, ?'TLS-1.X'=Version) when Version >= ?'TLS-1.2' ->
     {SignAlgo0, Param, _, _, _} = get_cert_params(Cert),
     SignAlgo = sign_algo(SignAlgo0, Param),
     is_acceptable_hash_sign(SignAlgo, HashSigns);
 is_acceptable_cert(_,_,_) ->
     %% Not negotiable pre TLS-1.2. So if cert is available for version it is acceptable
     true.
 
-supported_ecc({Major, Minor}) when ((Major == 3) and (Minor >= 1)) orelse (Major > 3) ->
-    Curves = tls_v1:ecc_curves(Minor),
+%% TODO: this function is not used. Can it be removed or does it need
```

Comment:
I think we can and should remove it. I think it the use of it probably got refactored out and was left here by accident!